schemas/predefined_plans.go

```diff
@@ -0,0 +1,98 @@
+package schemas
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Peripli/service-manager/pkg/instance_sharing"
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/gofrs/uuid"
+	"time"
+)
+
+func BuildReferencePlanSchema() string {
+	return fmt.Sprintf(`{
+  "name": "%[1]s",
+  "catalog_name": "%[1]s",
+  "description": "%[2]s",
+  "bindable": true,
+  "ready": true,
+  "metadata": {
+    "supportedPlatforms": []
+  },
+  "schemas": {
+    "service_instance": {
+      "create": {
+        "parameters": {
+          "$schema": "http://json-schema.org/draft-04/schema#",
+          "type": "object",
+          "additionalProperties": false,
```

Comment:
If we want the users  to add the additional label properties using the JSON display, then we should set additionalProperties=true, otherwise the validation in the cockpit will block this

Reply:
we will keep it false for now as we don't support any other keys.